### PR TITLE
Cli dds documentation

### PR DESCRIPTION
**Pull request by michaelbabyn**
resolves plotly/streambed#11881

Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

**Comment by michaelbabyn** [comment]
cc @ycaokris

File: tutorial/dash_deployment_server_examples.py [hunk 1]
```diff
@@ -1537,6 +1537,421 @@ def display_instructions(platform):
...
+    '''),
+
+    dcc.Markdown('''
+
+    ### List of exposed DDS commands:
+
+
+    #### App-related Commands:
+    
+    > These commands can only be run the app-owner.'''),
```
Comment (ycaokris) [inline tutorial/dash_deployment_server_examples.py]:
```suggestion
    > These commands can only be run by the app-owner.'''),
```

File: tutorial/dash_deployment_server_examples.py [hunk 2]
```diff
@@ -1537,6 +1537,421 @@ def display_instructions(platform):
...
+
+    html.Details([
+        html.Summary('Unlock app'),
+        dcc.Markdown('''
+        &nbsp;
+        
+        In some cases, it may be necessary to remove an existing deploy lock.
+        This can be performed via the apps:unlock command.
+        
+        > **Warning**: Removing the deploy lock will not stop in progress deploys.
```
Comment (ycaokris) [inline tutorial/dash_deployment_server_examples.py]:
```suggestion
        > **Warning**: Removing the deploy lock will not stop in-progress deploys.
```

File: tutorial/dash_deployment_server_examples.py [hunk 3]
```diff
@@ -1537,6 +1537,421 @@ def display_instructions(platform):
...
+        At this time, in-progress deploys will need to be manually terminated by
+        someone with server access.
```
Comment (ycaokris) [inline tutorial/dash_deployment_server_examples.py]: Does this means someone who has admin access?

File: tutorial/dash_deployment_server_examples.py [hunk 4]
```diff
@@ -1537,6 +1537,421 @@ def display_instructions(platform):
...
+    
+        **Example:**
+        `ssh dokku@your-dash-deployment-server -p PORT apps:unlock my-dash-app`
+        
+        &nbsp;
+        # ''')]),
+
+    html.Details([
+        html.Summary('Get app logs'),
```
Comment (ycaokris) [inline tutorial/dash_deployment_server_examples.py]: 👍  This is really useful!

File: tutorial/dash_deployment_server_examples.py [hunk 5]
```diff
@@ -1537,6 +1537,421 @@ def display_instructions(platform):
...
+            
+            &nbsp;
+        ''')
+    ]),
+
+    dcc.Markdown('''
+
+    #### Service-related Commands:
+
+    > These commands can only be run the service-owner.'''),
```
Comment (ycaokris) [inline tutorial/dash_deployment_server_examples.py]:
```suggestion
    > These commands can only be run by the service-owner.'''),
```

Comment (ycaokris) [reply tutorial/dash_deployment_server_examples.py]: what's the difference between service owner and app owner?

**Comment by ycaokris** [comment]
Looks like review app fails on heroku. @michaelbabyn  could you please double-check?

**Comment by michaelbabyn** [comment]
@ycaokris ,  I think the heroku deployment worked when I initially made this PR but it's not around anymore because the PR is old. I can run the docs locally in a VM configured to use the `dash-docs` `requirements.txt`/`runtime.txt` files so I don't think it's broken/ would fail on heroku.

**Pull request merged**